pyproject.toml

```diff
@@ -13,11 +13,11 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.9"
 cdislogging = "1.1.1"
-cryptography = "^37.0.4"
-dictionaryutils = "^3.0.2"
+cryptography = ">=41.0.5,<42"
```

Comment:
Chris asked us [here](https://ctds-planx.atlassian.net/browse/PPS-766?focusedCommentId=31219) to update to cryptography version 42.0.2, was this not possible?